src/main/resources/assets/minecraft/texts/credits.json

```diff
@@ -4429,7 +4429,7 @@
                     "title": "Translators",
                     "names": [
                         "Blurryface (ɥsᴉʅƃuƎ, Brazilian Portuguese)",
-                        "Fery (Spanish)",
+                        "NormalFery 'Fery' (Spanish)",

```

Comment:
For future notice, this section doesn't need to follow Alphabetical Order.

Reply:
it doesnt here but if someone wants to change their position to follow the order i can link to this so they know they dont need to